src/app/ApplicationTemplate.Shared.Views/Framework/Email/EmailService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System.Threading.Tasks;
+
+namespace ApplicationTemplate;
+
+public sealed partial class EmailService : IEmailService
+{
+	public async Task Compose(Email email)
+	{
+		await PlatformCompose(email);
+
+//-:cnd:noEmit
+/* TODO: Use Uno implementation for when attachments are supported.
+ * See https://github.com/unoplatform/uno/issues/2432 for more details. */
+#if false
```

Comment:
Wouldn't this code work for Windows though?

Reply:
Yes but it won't send any attachments. Right now I've tested Android and it's working with Essentials copied code. My last commit contain some incomplete stuff that I had to push to not lose anything.